docs/sdkx-cocos2dx/notifications-ios.mdx

```diff
@@ -0,0 +1,263 @@
+
+Example usage:
+
+When receiving remote notifications when the app starts for the first time, or after being killed by the user:
+
+```objc
+#import <HelpshiftX/Helpshift.h>
+...
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
+{
+    ...
+    if(launchOptions[UIApplicationLaunchOptionsRemoteNotificationKey]) {
+        NSDictionary *userInfo = launchOptions[UIApplicationLaunchOptionsRemoteNotificationKey];
+        if([[userInfo objectForKey:@"origin"] isEqualToString:@"helpshift"]) {
+            [Helpshift handleNotificationWithUserInfoDictionary:notification.request.content.userInfo
+                                                    isAppLaunch:YES
+                                                 withController:self.window.rootViewController];
+        }
+    }
+    ...
+}
+
+```
+
+For the `didReceiveRemoteNotification:` delegate -
+
+```objc
+#import <HelpshiftX/Helpshift.h>
+...
+- (void) application:(UIApplication *)application didReceiveRemoteNotification:(NSDictionary *)userInfo {
+    if([[userInfo objectForKey:@"origin"] isEqualToString:@"helpshift"]) {
+        [Helpshift handleNotificationWithUserInfoDictionary:notification.request.content.userInfo
+                                                isAppLaunch:NO
+                                             withController:self.window.rootViewController];
+    }
+}
+```
+
+## In-app notifications {#in-app-notifications}
+
+In-app notifications are similar to Apple's push notification banners. Unlike push notifications, they appear within your app when it is in use by the user.
+
+These notifications are sent when an agent replies to a customer's issue. Your customers can click on these banners to go straight into the conversation screen.
+
+<SideBySide>
+
+<Image
+  src="/static/books/sdkx_ios/In-app-notification.png"
+  width="half"
+  alt="In-app-notification.png"
+/>{" "}
+
+</SideBySide>
+
+## Configuring In-app notifications {#optional-flags}
+
+The Helpshift [install call](/sdkx-cocos2dx/getting-started#initializing) supports flags for configuring SDK behaviour.
+Currently we support one flag i.e `enableInAppNotification`.
+
+### Enabling/Disabling In-app notifications {#enable-disable-in-app}
+
+<div className="compact ">
+
+|         |                         |
+| ------- | ----------------------- |
+| Flag    | enableInAppNotification |
+| Values  | true/false              |
```

Comment:
Use YES/NO rather than true/false here - or change the YES/NO in description that follows to true/false.